package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "todo-list-",
+  "version": "1.0.0",
+  "description": "To-do list\" is an application that allows a user to keep track of tasks they need to do and allows the user to mark them as complete and clear them from the UI once they are done.",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
```

Comment:
- [ ] **[OPTIONAL]** I think it would be nice to include `npm start` as a viable option for starting the app.

  Simply install dev server by running `npm install -D webpack-dev-server`, then add `"start": "webpack serve --open"` to this scripts object.